openapi/components/schemas/GatewayAccountConfig/OnRamp.yaml

```
@@ -14,3 +14,11 @@ allOf:
             format: password
         required:
           - apiKey
+      settings:
+        type: object
+        description: OnRamp settings object.
+        properties:
+          useServerToServer:
```

```suggestion
          useServerToServerApi:
```


https://github.com/Rebilly/api-definitions/blob/200b5f100115aee839b7a17e5e41d5379fb2cc36/openapi/components/schemas/GatewayAccountConfig/AstroPayCard.yaml#L35

?